### DISCUSSION TITLE
Sub menu: Export as ..?

### DISCUSSION THREAD

**Discussion by seghier** [Comfy-Org/ComfyUI#5338] [Q&A]
Hello
How to add sub-menu to export a custom format of the workflow ( to python)?

![image](https://github.com/user-attachments/assets/71f1a640-46b5-47af-b5d5-d7bb8f7058cb)

**Comment by ltdrdata** [comment]
That is not a built-in feature of ComfyUI.

**Comment by seghier** [comment]
Thanks @ltdrdata 
So it is not possible to add custom menu ; buttons ?

**Reply by ltdrdata** [reply]
Are you a custom node developer?

**Reply by seghier** [reply]
I still learn about nodes in Comfyui
I am plugin developer for Rhino3d and Grasshopper
the plugin load json file and convert it to Python but i prefer convert it directly from Comfyui, so i need to add sub menu or button to make this possible
![image](https://github.com/user-attachments/assets/3eb988e9-41c8-4f10-b5e9-0339e696b87f)
